### PR TITLE
Fix handling of utf8 offsets for text analyzer

### PR DESCRIPTION
**Pull request by gnusi**

### CODE REVIEW

File: core/utils/wildcard_utils.cpp [hunk 1]
```diff
@@ -59,7 +58,7 @@ WildcardType wildcard_type(bytes_ref expr) noexcept {
   const auto* end = expr.end();
 
   for (size_t i = 0; char_begin < end; ++i) {
-    const auto char_length = utf8_utils::cp_length(*char_begin);
+    const size_t char_length = utf8_utils::cp_length_msb(*char_begin);
```
Comment (MBkkt) [inline core/utils/wildcard_utils.cpp:61]: What means msb? Why auto -> size_t

File: core/utils/wildcard_utils.cpp [hunk 2]
```diff
@@ -131,7 +128,7 @@ automaton from_wildcard(bytes_ref expr) {
   const auto* end = expr.end();
 
   while (label_begin < end) {
-    const auto label_length = utf8_utils::cp_length(*label_begin);
+    const auto label_length = utf8_utils::cp_length_msb(*label_begin);
```
Comment (MBkkt) [inline core/utils/wildcard_utils.cpp:131]: why here not size_t?

File: core/analysis/text_token_stream.cpp [hunk 3]
```diff
@@ -1072,18 +1097,37 @@ bool text_token_stream::next() {
 
 bool text_token_stream::next_word() {
```
Comment (MBkkt) [inline core/analysis/text_token_stream.cpp:1098]: Only this function contains not formatting changes?

**Review by MBkkt — APPROVED** [review]

**Review by Dronplane — APPROVED** [review]
LGTM

**Pull request merged**